src/components/ui/toolbox.ts

```diff
@@ -108,6 +109,12 @@ export default class Toolbox extends EventsDispatcher<ToolboxEvent> {
    */
   private clickListenerId: string = null;
 
+  /**
+   * Stores popover height.
+   * Allows not to recalculate height on each popover opening
+   */
+  private popoverHeightCached: number | null = null;
```

Comment:
you don't need this. We have a special `cacheable` decorator for that reason.

Try something like this:

<img width="974" alt="image" src="https://user-images.githubusercontent.com/3684889/163708888-d30adf0e-224b-40f8-92f0-62f5446ee78a.png">

Reply:
fixed